src/api/handlers/member.ts

```diff
@@ -276,3 +276,17 @@ export async function editBotProfile(username?: string, botAvatarURL?: string) {
     },
   );
 }
+
+/** Edit the nickname of the bot in this guild */
+export async function editBotNickname(
+  guildID: string,
+  nickname?: string | null,
+) {
+  const hasPerm = await botHasPermission(guildID, ["CHANGE_NICKNAME"]);
+  if (!hasPerm) throw new Error(Errors.MISSING_CHANGE_NICKNAME);
+
+  return (await RequestManager.patch(
```

Comment:
No `return await`s

Reply:
I did that because I wanted to return the new nickname string. I remember there are a few other functions doing that.

Reply:
Feel free to open a PR changing that.

Reply:
Rn this function returns the new nickname string. Shall I remove that and add that back in a new PR, do you mean that?

Reply:
@itohatweb assign it to a new variable and return that variable.